certified-connectors/dvelop/apiDefinition.swagger.json

```diff
@@ -835,6 +835,474 @@
           }
         }
       }
+    },
+	"/default/updatedocument/execute": {
+      "post": {
+        "tags": [
+          "Default actions"
+        ],
+        "summary": "Update document",
+        "description": "Update a document in d.velop documents.",
+        "operationId": "UpdateDocument",
+        "x-ms-visibility": "important",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "repositoryId",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Select an repository of your d.velop documents environment.",
```

Comment:
@BeritHorst please make sure that everything is grammatically correct and is free of typographical errors. Please refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).